src/components/Select.js

```diff
@@ -66,16 +67,25 @@ const Select = React.createClass({
     if (!isMobile) {
       this.setState({
         isOpen: !this.state.isOpen
+      }, () => {
+        if (document.getElementById('items')) {
+          const items = document.getElementById('items');
+
+          items.scrollTop = this.state.scrollLocation;
+        }
```

Comment:
I know this is weird to do a callback in `setState` but I haven't figured out a good way to get around it.  If i  do something like this:
```javascript
_handleClick () {
  if (!isMobile) {
    this.setState({
      isOpen: !this.state.isOpen
    });
  }

  this.setScrollPosition();
},

setScrollPosition () {
  if (document.getElementById('items')) {
    const items = document.getElementById('items');

    items.scrollTop = this.state.scrollLocation;
  }
},
```
this.state.scrollLocation` is not set to the proper value when the select component is closed, and thus it won't fix the problem